common/test/resources/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can we put this under common/web/test (where 'web' is roughly equivalent to js/ts these days so perhaps we'll rename to 'js' one day in the future)?

Reply:
... so, move the entire folder?  There are .kmn keyboards and such in there too.

Granted, I did note that there's `common/test/keyboards` as well as `common/test/resources/keyboards`... but I think the intent is for us to be able to keep these test resources in a common place not tied to just the .js.  My thought is that the former should merge with the latter... but of course, that's _my_ thought.

Reply:
I think only the keyboards would be useable by other products for testing. For now, I think it makes sense to move the entire `resources` folder under `common/web/test`.

You are right: I do have a lowpri [issue](https://github.com/keymanapp/keyman/issues/5815) to organise all the keyboards; I'm thinking it's also fairly low value -- the keyboard sources have in many cases diverged to support specific test cases, so trying to have a unified set of keyboards may end up being more hassle than its worth.